xwiki-rendering-api/src/main/java/org/xwiki/rendering/internal/transformation/DefaultTransformationManager.java

```diff
@@ -79,6 +79,15 @@
     @Named("context")
     private Provider<ComponentManager> componentManagerProvider;
 
+    /**
+     * {@inheritDoc}
+     * inherits on {@link TransformationManager}
+     *
+     * @deprecated
+     *
+     * Replaced by {@link #performTransformations(Block, TransformationContext)}
```

Comment:
Should be located in the same line that `@deprecated` (this is the documentation of the `@deprecated` tag)